mypy/private/mypy.py

```diff
@@ -1,12 +1,11 @@
+import argparse
 import contextlib
 import pathlib
 import shutil
 import sys
 import tempfile
 from typing import Any, Generator
 
-import click
```

Comment:
getting rid of the click dependency makes it easier for a user to override the mypy requirement without also having to supply click to rules_mypy.